frontend/src/components/AppTable.vue

```diff
@@ -454,21 +454,6 @@ const ariaSort = computed(() => {
   text-transform: capitalize;
 }
 
-/* first th */
-.th:nth-child(1) {
-  width: 30%;
-}
-
-/* second th */
-.th:nth-child(2) {
-  width: 5%;
-}
-
-/* third th */
-.th:nth-child(3) {
-  width: 30%;
-}
-
 /** body cells */
```

Comment:
This is where I might be causing trouble by altering potentially broader impact css, but I _think_ I'm actually reducing the trouble that this overly broad css was causing. This PR is the first of many for defining the columns for each association type separately, and it doesn't make sense that we'll always want these widths. Plus, if you scan through Human Disease on this branch, most tables look better, in particular it gives as much space as possible to disease models.